### PR TITLE
PLANET-5972  Prevent using PNG images for photographs

### PR DESCRIPTION
**Pull request by oekeur**
[P4 Repo issue](https://github.com/greenpeace/planet4/issues/135)
[JIRA Issue](https://jira.greenpeace.org/browse/PLANET-5972)

This PR limits the ability of selecting/uploading PNG on certain blocks and provides pointers to editors about this restriction.

Carousel header
-[X] Limits selection to jpeg/jpg only
-[X] Adds ``(only jpeg/jpeg)`` to the media library modal
image

Gallery
~~-[ ] Limits upload to jpg/jpeg only~~ _In the current implementation this doesn't seem possible ATM_
-[X] Limits selection to jpg/jpg only
-[X] Adds ``jpeg`` to the media placeholder text
image

### CODE REVIEW

File: assets/src/blocks/Gallery/GalleryEditor.js [hunk 1]
```diff
@@ -67,11 +67,11 @@ const renderEdit = (attributes, setAttributes) => {
           addToGallery={hasImages}
           labels={{
             title: __('Select Gallery Images', 'planet4-blocks-backend'),
-            instructions: __('Upload an image or select from the media library.', 'planet4-blocks-backend'),
+            instructions: __('Upload an JPEG image or select one from the media library.', 'planet4-blocks-backend'),
```
Comment (GP-Dan-Tovbein) [inline assets/src/blocks/Gallery/GalleryEditor.js:70]: I think that "JPG" maybe sounds better

**Review by GP-Dan-Tovbein — APPROVED** [review]
Code looks good to me!

Just a small question, why don't use PNG?

**Comment by sagarsdeshmukh** [comment]
Hi Dan,
> Just a small question, why don't use PNG?

( From JIRA ticket desc)

> We currently allow to use PNG images in a lot of places, but these cannot be (lossy) compressed by Cloudflare so end up being more than 10 times as big for the same image. We can restrict the images that can be chosen by file type so that they can only pick a JPG for blocks that are supposed to show photographs (e.g. carousel).

> For a sense of scale of the impact. Just a single png photograph on the page can drop the pagespeed score by more than 10 points out of 100.

**Comment by GP-Dan-Tovbein** [comment]
> Hi @GP-Dan-Tovbein ,
> 
> > Just a small question, why don't use PNG?
> 
> ( From JIRA ticket desc)
> 
> > We currently allow to use PNG images in a lot of places, but these cannot be (lossy) compressed by Cloudflare so end up being more than 10 times as big for the same image. We can restrict the images that can be chosen by file type so that they can only pick a JPG for blocks that are supposed to show photographs (e.g. carousel).
> 
> > For a sense of scale of the impact. Just a single png photograph on the page can drop the pagespeed score by more than 10 points out of 100.

Oh, thanks @sagarsdeshmukh ! Sorry, I haven't checked the ticket 🤦 , only the code.

**Comment by comzeradd** [comment]
Thanks @oekeur 💚 

I merged it offline at d29403bc9c9fcb8a1b01305e85bae11e7c49057f just to squash and make tests happy.

**Pull request closed without merging**